### ISSUE TITLE
Notification delay

### ISSUE THREAD

**Issue by YanDoroshenko** [Toqozz/wired-notify#113]
There's a small delay (~200ms) between an action triggering a notification and the notification itself. Most noticeable when changing volume. Doesn't happen with xfce4-notifyd.

Wired version: 0.10.2.

Maybe it has something to do with polling interval?

**Comment by biopsin** [comment]
Im noticing this too with last release, Ive built from master but not yet tested..

**Comment by Toqozz** [comment]
Yep, this is very likely a poll interval thing.  I haven't tested this, but it probably comes from `dbus.rs`:
```rust

pub fn process_dbus() {
    let conn = get_connection();
    loop {
        match conn.process(Duration::from_millis(1000)) {
            Ok(_) => (),
            Err(e) => eprintln!("DBus Error: {}", e),
        }
    }
}
```

I've been meaning to make that configurable.

**Comment by Toqozz** [comment]
Sorry, it's actually this one: https://github.com/Toqozz/wired-notify/blob/f4300acba8180f8fc6babc1a118f9b3753a2b052/src/main.rs#L164

Adding a config option now.

**Comment by Toqozz** [comment]
Fixed in `73bb268`.  See `idle_poll_interval` option in [`wired.ron`](https://github.com/Toqozz/wired-notify/blob/73bb26863ed3272e0d524424685bd53a70db25e4/wired.ron#L17).
